activities/tool-management/github.md

```diff
@@ -0,0 +1,21 @@
+---
+type: resource
+---
+
+# GitHub
+
+GitHub, Inc. is a subsidiary of Microsoft which provides hosting for software development and version control using Git. It offers the distributed version control and source code management (SCM) functionality of Git, plus its own features. It provides access control and several collaboration features such as bug tracking, feature requests, task management, continuous integration and wikis for every project.
+
+## Using GitHub
+
+GitHub is our bread and butter when it comes to codebase management, codebase auditing, issue tracking and version control, both for ourselves and our community.
+
+## GitHub management
+
+GitHub is managed by the Directors and Codebase Stewards. New employees are onboarded by one of the Codebase Stewards. We also have [documention available on About for newcomers](../trainings/github-for-newcomers.md).
+
+## Privacy and GDPR
+
+GitHub has [an extensive Privacy Notice.](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-privacy-statement)
+
+> "GitHub collects information directly from you for your registration, payment, transactions, and user profile. We also automatically collect from you your usage information, cookies, and device information, subject, where necessary, to your consent. GitHub may also collect User Personal Information from third parties. We only collect the minimum amount of personal information necessary from you, unless you choose to provide more."
```

Comment:
This adds the paragraph for the other info I was missing - I think this will also be ready to go then!
```suggestion
> "GitHub collects information directly from you for your registration, payment, transactions, and user profile. We also automatically collect from you your usage information, cookies, and device information, subject, where necessary, to your consent. GitHub may also collect User Personal Information from third parties. We only collect the minimum amount of personal information necessary from you, unless you choose to provide more."

"GitHub processes personal information both inside and outside of the United States and relies on Standard Contractual Clauses as the legally provided mechanism to lawfully transfer data from the European Economic Area, the United Kingdom, and Switzerland to the United States. In addition, GitHub is certified to the EU-US and Swiss-US Privacy Shield Frameworks."
```